### DISCUSSION TITLE
The code in the document doesn't work.

### DISCUSSION THREAD

**Discussion by ohanamaru** [react-hook-form/react-hook-form#7732] [Q&A] [answered]
I am running the code at the end of the following link, MULTIPLE ERROR MESSAGES↓
https://react-hook-form.com/api/useformstate/errormessage

However, it gave me an error and did not work.

`
const { register, formState: { errors }, handleSubmit } = useForm({
    criteriaMode "all"
  });
`

I modified the above code to set criteriaMode: "all" and it worked.

I think this code should be fixed.
Is there any way I can become a contributor myself and fix this?

I'm not very knowledgeable about github, but thank you for your help.

**Comment by Moshyfawn** [comment, marked as answer]
What's the issue you're facing?
Looks fine when I try it

https://user-images.githubusercontent.com/16290753/152326704-4bd1d532-6e97-4dfa-b52e-42faee978d00.mp4

**Reply by ohanamaru** [reply]
I copied and pasted the MULTIPLE ERROR MESSAGES into the react app.
Then I get the following error.

```
Failed to compile.

SyntaxError: /Users/program/src/components/pages/SignUp.js: Unexpected token, expected "," (7:17)
   5 | export default function App() {
   6 |   const { register, formState: { errors }, handleSubmit } = useForm({
>  7 |     criteriaMode "all"
     |                  ^
   8 |   });
   9 |   const onSubmit = data => console.log(data);
  10 |
```

**Reply by bluebill1049** [reply]
nothing stops you from sending us a PR to improve the doc :) it's very easy as well:  https://github.com/react-hook-form/documentation/edit/master/src/data/en/api.tsx edit it and send the PR and get your name into the contribution list 👯

**Reply by Moshyfawn** [reply]
Ah, you're talking about the missing : _(colon)_?
You can edit [this file](https://github.com/react-hook-form/documentation/blob/master/src/components/codeExamples/errorsMessageTs.ts) by
- clicking
<img width="312" alt="image" src="https://user-images.githubusercontent.com/16290753/152330231-71f24eaf-8de5-4ddc-810e-52bc0ad33e49.png">

- editing the file

- submitting the PR
<img width="453" alt="image" src="https://user-images.githubusercontent.com/16290753/152330330-92685cb5-c0fb-4879-bba9-3f3c221ae9d9.png">

**Reply by ohanamaru** [reply]
Very kind of you, thank you!
I've made a pull request for a different account!
I hope it will be approved (^^)
Thank you very much!

https://github.com/react-hook-form/documentation/pull/725

**Reply by ohanamaru** [reply]
Thank you for approving my pull request!
This is my very first OSS development!😁💗

**Reply by bluebill1049** [reply]
Awesome work! Thank you for your contribution. Now your name stays with this OSS! ✨

**Reply by bluebill1049** [reply]
oh and don't hesitate to send more as well 🙏
